src/mcts/params.cc

```diff
@@ -283,7 +285,11 @@ const OptionId SearchParams::kWDLRescaleDiffId{
     "Shifts the logistic WDL mean by diff in white's favor."};
 const OptionId SearchParams::kWDLContemptId{
     "wdl-contempt", "WDLContempt",
-    "The simulated rating advantage for the WDL conversion."};
+    "The simulated rating advantage for the WDL conversion. Comma separated "
+    "list in the form [name=]value, where the name is compared with the "
+    "`UCI_Opponent` value to find the appropriate contempt value. The default "
+    "value is taken from `UCI_RatingAdv` and will be overriden with a value "
+    "given here without the name part."};
```

Comment:
Can this be understood in the intended way? `UCI_RatingAdv` will be overridden if either a value without name is given, or if a name match is found.